src/composables/use-hide-on-click-outside.js

```diff
@@ -37,9 +37,7 @@ function useMouseDownTargetRef({
       const document = getDocument(popover)
       const onMouseDown = (event) => (mouseDownTargetRef.value = event.target)
       document.addEventListener('mousedown', onMouseDown)
-      onInvalidate(() => {
-        document.addEventListener('mousedown', onMouseDown)
-      })
+      onInvalidate(() => {})
```

Comment:
Seems like there was a bit of a misunderstanding of the issue here :sweat_smile: The issue says that we need to remove the event listener, but only because we've typo'd this `onInvalidate` callback to call `addEventListener` _again_ for an event that's been added. What we meant to do was call `removeEventListener`. Easy mistake to make :slightly_smiling_face: 

```suggestion
      onInvalidate(() => {
        document.removeEventListener('mousedown', onMouseDown)
      })
```

Reply:
Oops, Olga got to it a second before me :slightly_smiling_face:

Reply:
I'm not sure who was first 😆

Reply:
Whoops! Fixing it now, thank you!